config/routes

```diff
@@ -10,7 +10,6 @@
 /packages/#PathPackageName/#PathVersion/docs           PackageVersionDocsR         GET
 /packages/#PathPackageName/#PathVersion/docs/#Text     PackageVersionModuleDocsR   GET
 !/packages/#PathPackageName/docs/#Text                 PackageModuleDocsR          GET
-!/packages/#PathPackageName/badge                      PackageBadgeR               GET
```

Comment:
I'm not actually sure we can drop badge support altogether. If I recall correctly, most of the packages within core make use of this for their READMEs. If we need to fork `badger` to work with 9.2.3 then I could probably pick up that task.

Reply:
Looks like it just needs its upper bound updated: https://github.com/PureFunctor/barrier/commit/79e8fc36fb7d36591f9f65c0e0eb892d8062cdc5

Reply:
cool, yeah that sounds doable.

Reply:
Looks like Phil did the same thing a few years ago: https://github.com/philopon/barrier/pull/4

Reply:
ok I guess we can try open a PR and see if we get a response

Reply:
Ok I undid the badge removal and added @PureFunctor 's fork/commit as a local dependency. This builds for me using cabal. I'm not sure if it works on stack, as I cannot get stack to run. Would be great if one of you could test it.
Or do we actually need/use stack?

Reply:
Yeah, we need to specify a git dependency for it under `extra-deps`. I don't have a `stack` built at hand at the moment, since it's pretty wonky at M1. I will transfer over my fork to the `purescript` organization in the meantime as well.

Reply:
I added it to extra-deps, but I can't get it to build.

Reply:
Looking into it...

Reply:
Ok, I think I need to relax the bounds for `bytestring` in `barrier`: PureFunctor/barrier@db5e27c

Reply:
Thanks to @i-am-the-slime I got the stack build working as well.

Reply:
@PureFunctor if you move it to `purescript-contrib` I can update the dependencies. Apart fromt his I think it is good to go now